src/QuantumSdk/README.md

```diff
@@ -109,6 +109,9 @@ May contain additional arguments to pass to the Q# command line compiler. Valid
 - `CsharpGeneration`:    
 Specifies whether to generate C# code as part of the compilation process. Setting this property to false may prevent certain interoperability features or integration with other pieces of the Quantum Development Kit. 
 
+- `DefaultSimulator`:
+Specifies the simulator to use by default for execution. 
```

Comment:
I added "Valid values are QuantumSimulator, ToffoliSimulator, ResourcesEstimator, or the fully qualified name of a custom simulator. ". 
It would be good to add a test for a custom simulator in your PR.